easybuild/easyblocks/c/comsol.py

```diff
@@ -59,7 +59,7 @@ def extract_step(self):
     def configure_step(self):
         """Configure COMSOL installation: create license file."""
 
-        default_lic_env_var = 'LMCOMSOL_LICENSE_FILE'
+        default_lic_env_var = 'EB_COMSOL_LICENSE_FILE'
```

Comment:
Can't we consider both?

Need to check whether `find_flexlm_license` considers all variables passed to `custom_env_vars` (but I think it does)

Reply:
i just checked and it does consider all of them indeed